pytorch_lightning/trainer/training_io.py

```diff
@@ -119,6 +119,15 @@
 else:
     HOROVOD_AVAILABLE = True
 
+try:
+    import omegaconf
+except ImportError:
+    OMEGACONF_AVAILABLE = False
+else:
+    OMEGACONF_AVAILABLE = True
+
```

Comment:
Can you remove these extra whitelines? Should only be 2 between imports and the class def